app/src/main/java/eu/kanade/presentation/more/settings/screen/advanced/ClearDatabaseScreen.kt

```diff
@@ -90,7 +95,20 @@ class ClearDatabaseScreen : Screen() {
                             }
                         },
                         text = {
-                            Text(text = stringResource(MR.strings.clear_database_confirmation))
+                            Column {
+                                Text(
+                                    text = if (keepReadManga) {
+                                        stringResource(MR.strings.are_you_sure)
+                                    } else {
+                                        stringResource(MR.strings.clear_database_confirmation)
+                                    },
+                                )
+                                LabeledCheckbox(
+                                    label = stringResource(MR.strings.clear_db_exclude_read),
+                                    checked = keepReadManga,
+                                    onCheckedChange = { keepReadManga = it },
+                                )
```

Comment:
"Are you sure?" should be moved into title

Then here first there should be a text "You’re about to remove entries from the database"
Then a switch instead of checkbox "Keep entries with read chapters"
Underneath the switch if it's disabled a small warning "Read chapters and progress of non-library entries will be lost"